README.md

```diff
@@ -22,9 +21,8 @@ To get the project running you will need to:
 ### Prerequisites
 
 * An IDE supporting .NetCore 3.1
-* The [Tools Service](https://github.com/SkillsFundingAgency/das-tools-service)
-* A valid GitHub account that is a member of the SkillsFundingAgency organisation
-* The "Support Portal" role added to your GitHub account
+* A valid Staff IDAMS account
+* The SCS role added to your IDAMS account
```

Comment:
There a multiple roles, SCS and SCP. If you mention one you may as well mention both?

Reply:
SCP is not used at the moment, that is why I have not included it